internal/test/wmcb/wmcb_test.go

```diff
@@ -64,20 +66,21 @@ var (
 	// filesToBeTransferred holds the list of files that needs to be transferred to the Windows VM
 	filesToBeTransferred = flag.String("filesToBeTransferred", "",
 		"Comma separated list of files to be transferred")
-	// kubeNode contains the information about  the kubernetes node package for Windows
-	kubeNode = pkgInfo{
-		url:     "https://dl.k8s.io/v1.16.2/kubernetes-node-windows-amd64.tar.gz",
-		sha:     "a88e7a1c6f72ea6073dbb4ddfe2e7c8bd37c9a56d94a33823f531e303a9915e7a844ac5880097724e44dfa7f4a9659d14b79cc46e2067f6b13e6df3f3f1b0f64",
-		shaType: "sha512",
-	}
+	// kubeNodePkgName is the user-defined name of the required kube node package
+	kubeNodePkgName = pkgName("kubeNode")
+	// cniPluginPkgName is the user-defined name of the required cni plugins package package
+	cniPluginPkgName = pkgName("cniPlugins")
```

Comment:
Nit: This looks like overkill but is most likely @ravisantoshgudimetla's suggestion ;-) 

Not a nit: This needs to go into the previous commit.